src/helpers/curve.ts

```diff
@@ -142,14 +134,11 @@ export function getAmountBFromLiquidityDelta(
   minSqrtPrice: BN,
   rounding: Rounding
 ): string {
-  const delta = currentSqrtPrice.sub(minSqrtPrice);
-  // Q64.64 * Q64.64 => prod: Q128.128
-  const prod = new Decimal(liquidity.toString()).mul(
-    new Decimal(delta.toString())
-  );
-
+  const one = new BN(1).shln(128);
+  const denominator = currentSqrtPrice.sub(minSqrtPrice);
+  const result = liquidity.mul(denominator); // Q128
```

Comment:
denominator can only be named if you do division, pls use other variable if you are using mul

Reply:
updated